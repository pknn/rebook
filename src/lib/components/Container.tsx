import React from 'react';

const Container: React.FunctionComponent = ({ children }) => {
  return <div className="container">{children}</div>;
};

export default Container;
