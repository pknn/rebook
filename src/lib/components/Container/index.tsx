import React from 'react';

const Container: React.FC = ({ children }) => {
  return <div className="container flex mt-16"> {children} </div>;
};

export default Container;
