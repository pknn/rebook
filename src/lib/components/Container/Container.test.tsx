import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

import Container from './index';

const mockChildren: JSX.Element = <h1>Hello</h1>;

describe('<Container />', () => {
  it('should render correctly', () => {
    const component = shallow(<Container>{mockChildren}</Container>);

    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
