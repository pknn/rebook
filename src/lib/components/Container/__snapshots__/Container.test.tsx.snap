// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render correctly 1`] = `
<div
  className="container flex mt-16"
>
   
  <h1>
    Hello
  </h1>
   
</div>
`;
